Mathlib/Order/Filter/Prod.lean

```diff
@@ -378,6 +378,9 @@ theorem prod_pure {b : β} : f ×ˢ pure b = map (fun a => (a, b)) f := by
 theorem prod_pure_pure {a : α} {b : β} :
     (pure a : Filter α) ×ˢ (pure b : Filter β) = pure (a, b) := by simp
 
+theorem principal_prod_principal (a : α) (b : β) : 𝓟 {a} ×ˢ 𝓟 {b} = 𝓟 {(a, b)} := by
```

Comment:
Do you need this lemma? Its `simp` normal form is `prod_pure_pure`. Also, `principal_prod_principal` should be about the product of `principal s`, not specific to singletons (I think that `prod_principal_principal` should be renamed to `principal_prod_principal`).

Reply:
Ah! Yes, that makes more sense

Reply:
I suggest that we drop this lemma for now.